react-app/src/utils/crumbs.test.ts

```diff
@@ -0,0 +1,128 @@
+// crumbs.test.ts
+import { describe, it, expect, vi } from 'vitest';
+import {
+  getDashboardTabForAuthority,
+  detailsAndActionsCrumbs,
+  dashboardCrumb,
+  detailsCrumb,
+  actionCrumb,
+} from './crumbs';
+import { Action } from 'shared-types/actions';
+
+// Mocking `mapActionLabel` function
+vi.mock('@/utils', () => ({
+  mapActionLabel: (action: Action) => {
+    switch (action) {
+      case Action.RESPOND_TO_RAI:
+        return "Respond to Formal RAI";
+      case Action.WITHDRAW_RAI:
+        return "Withdraw Formal RAI";
+      default:
+        return "Unknown Action";
+    }
+  },
+}));
```

Comment:
I don't think this is necessary. Does the test work without it?

Reply:
yes, removed it